CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## 6.19.2 (November 11, 2022).
+
+IMPROVEMENTS:
+
+* resources/artifactory_keypair: add `passphrase` attribute to the JSON body. No API errors in Artifactory 7.41.13 and up. Issue: [#574](https://github.com/jfrog/terraform-provider-artifactory/issues/574)
+PR: [#581](https://github.com/jfrog/terraform-provider-artifactory/pull/581)
```

Comment:
Extra line break

Reply:
No need to do it here. The PR link will be on the same line in the UI.